examples/benchmark/tls_bench.c

```diff
@@ -423,18 +423,34 @@ static void* client_thread(void* args)
                 info->client_stats.txTotal += ret;
             }
 
-            /* read echo of message */
-            start = gettime_secs(1);
-            ret = wolfSSL_read(cli_ssl, buf, sizeof(buf)-1);
-            info->client_stats.rxTime += gettime_secs(0) - start;
-            if (ret > 0) {
-                info->client_stats.rxTotal += ret;
+            /* Allocate buf after handshake is complete */
```

Comment:
Would it make more sense to allocate this just after the connect and above the while loop? Then move the free to handle outsize the loop? Seems better to just allocate once since this value won't change post handshake. It looks like you do this in the server thread below.

Reply:
Yes, agreed.